tests/src/e2e.rs

```diff
@@ -445,6 +445,114 @@ mod tests {
         Ok(())
     }
 
+    /// In this test we:
+    /// 1. Run the ledger node
+    /// 2. Submit an invalid transaction
+    /// 3. Shut down the ledger
+    /// 4. Restart the ledger
+    #[test]
+    fn invalid_transactions() -> Result<()> {
+        let dir = setup();
+
+        let base_dir = tempdir().unwrap();
+        let base_dir_arg = &base_dir.path().to_string_lossy();
+
+        // 1. Run the ledger node
+        let mut cmd = Command::cargo_bin("anoman")?;
+        cmd.current_dir(&dir).env("ANOMA_LOG", "debug").args(&[
+            "--base-dir",
+            base_dir_arg,
+            "ledger",
+        ]);
+        println!("Running {:?}", cmd);
+        let mut session = spawn_command(cmd, Some(20_000))
+            .map_err(|e| eyre!(format!("{}", e)))?;
+
+        session
+            .exp_string("Anoma ledger node started")
+            .map_err(|e| eyre!(format!("{}", e)))?;
+        session
+            .exp_string("Started node")
+            .map_err(|e| eyre!(format!("{}", e)))?;
+
+        // 2. Submit a an invalid transaction
```

Comment:
```suggestion
        // 2. Submit a an invalid transaction (amount > source's balance)
```

Reply:
Thanks for adding a test case for this! You could also maybe add one that fails in `tx` (the code "2" above), e.g. use `$BERTHA` as the token address:
```rust
vec![
            "transfer",
            "--source",
            BERTHA,
            "--target",
            ALBERT,
            "--token",
            BERTHA,
            "--amount",
            "1_000.1",
        ];
```